package.json

```diff
@@ -42,43 +42,43 @@
     "babel-cli": "^6.26.0",
     "babel-eslint": "^7.2.3",
     "babel-jest": "^21.2.0",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "^7.1.4",
     "babel-plugin-react-intl": "^2.3.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "codecov": "^2.3.0",
-    "css-loader": "^0.28.5",
+    "css-loader": "^0.28.11",
     "enzyme": "^3.1.1",
     "enzyme-adapter-react-16": "^1.0.4",
     "eslint-config-edx": "4.0.0",
-    "extract-text-webpack-plugin": "^3.0.0",
+    "extract-text-webpack-plugin": "next",
```

Comment:
This package hasn't quite released support for webpack 4 yet, but I'm tracking it and will update this to a version number when it does.

Reply:
does this mean we're blocked on merging this?

answering my own question: nope, because `next` maps to `4.0.0-beta.0` according to https://www.npmjs.com/package/extract-text-webpack-plugin?activeTab=versions